src/platform/platform_posix.c

```diff
@@ -98,6 +101,17 @@ CxPlatSystemLoad(
     void
     )
 {
+    //
+    // There are a few cases where we might call CxPlatSystemLoad more than
+    // once. Let's guard against that. The caller is responsible for ensuring
+    // it's not called concurrently.
```

Comment:
Where? No other platforms protect against this, and I personally think it's a bug in the caller and not something we should protect against here (best case assert instead).

Reply:
In secnetperf, it's called twice. Once before we allocate msquic object. Once inside the object allocation.

Reply:
I assume you're using static linking of msquc?

Reply:
Oh, haha, you are right. But, it wouldn't to make this change to make things easier for static build, right? The binaries built not seeking the so file in the same folder was the reason I use static build.